inc/item_devices.class.php

```diff
@@ -926,6 +926,19 @@ function getTableGroup(CommonDBTM $item, HTMLTableMain $table, array $options,
                         $content = Dropdown::getDropdownName($dropdownType::getTable(), $link[$field]);
                         break;
 
+                     case 'progressbar':
+                        $percent = 0;
+                        if ($peer->fields[$attributs['full']] > 0) {
+                           $percent = round(100 * $this->fields[$field] / $peer->fields[$attributs['full']]);
+                        }
+                        $content = Html::progressBar("percent" . mt_rand(), [
+                           'create'  => true,
+                           'percent' => $percent,
+                           'message' => $this->fields[$field] . " ($percent %)",
```

Comment:
We already have a localized string for this
```suggestion
                           'message' => sprintf(__('%1$s (%2$d%%) '), $this->fields[$field], $percent),
```

Reply:
Also, a unit may be appreciate :)

Reply:
And you could use `Html::formatNumber($this->fields[$field])` to correclty handle thousand separator.

Reply:
done. Unit added in the header of the column